converters/fromZigbee.js

```diff
@@ -610,7 +619,12 @@ const converters = {
         cluster: 'msPressureMeasurement',
         type: ['attributeReport', 'readResponse'],
         convert: (model, msg, publish, options, meta) => {
-            const pressure = parseFloat(msg.data['measuredValue']);
+            let pressure = 0;
+            if (msg.data.hasOwnProperty('scaledValue') && msg.data.hasOwnProperty('scale')) {
```

Comment:
This will only work if the `scaledValue` and `scale` value is always send with the `msPressureMeasurement`. I would expect we need to have something similar to: https://github.com/Koenkk/zigbee-herdsman-converters/blob/f346bd72f1689348e7bc9e2e51247672d3c5addb/converters/fromZigbee.js#L713